From a10734d5f416cd5d42d5239ccc321c4de58f362b Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Date: Wed, 17 Feb 2021 14:29:10 +0700
Subject: [PATCH] recipes-demos: hmi: demo-videos:  update URLs of video files

Update URLs of video files due to changes on the Renesas website.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 recipes-demos/hmi/demo-videos_0.1.bb | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/recipes-demos/hmi/demo-videos_0.1.bb b/recipes-demos/hmi/demo-videos_0.1.bb
index ae10cb0..4226d95 100644
--- a/recipes-demos/hmi/demo-videos_0.1.bb
+++ b/recipes-demos/hmi/demo-videos_0.1.bb
@@ -1,11 +1,11 @@
 LICENSE = "CLOSED"
 DESCRIPTION = "Videos for demos"
 
-SRC_URI = "http://www.renesas.com/jp/ja/img/products/media/auto-j/microcontrollers-microprocessors/rz/rzg/hmi-mmpoc-videos/h264-fhd-60.mp4;name=video1 \
-           http://www.renesas.com/jp/ja/img/products/media/auto-j/microcontrollers-microprocessors/rz/rzg/hmi-mmpoc-videos/h264-wvga-30.mp4;name=video2 \
-           http://www.renesas.com/jp/ja/img/products/media/auto-j/microcontrollers-microprocessors/rz/rzg/hmi-mmpoc-videos/h264-bl10-fhd-30p-5m-aac-lc-stereo-124k-48000x264.mp4;name=video3 \
-           http://www.renesas.com/jp/ja/img/products/media/auto-j/microcontrollers-microprocessors/rz/rzg/hmi-mmpoc-videos/h264-vga-24.mp4;name=video4 \
-           http://www.renesas.com/jp/ja/img/products/media/auto-j/microcontrollers-microprocessors/rz/rzg/hmi-mmpoc-videos/h264-hd-30.mp4;name=video5 "
+SRC_URI = "http://resource.renesas.com/resource/lib/img/products/media/auto-j/microcontrollers-microprocessors/rz/rzg/hmi-mmpoc-videos/h264-fhd-60.mp4;name=video1 \
+           http://resource.renesas.com/resource/lib/img/products/media/auto-j/microcontrollers-microprocessors/rz/rzg/hmi-mmpoc-videos/h264-wvga-30.mp4;name=video2 \
+           http://resource.renesas.com/resource/lib/img/products/media/auto-j/microcontrollers-microprocessors/rz/rzg/hmi-mmpoc-videos/h264-bl10-fhd-30p-5m-aac-lc-stereo-124k-48000x264.mp4;name=video3 \
+           http://resource.renesas.com/resource/lib/img/products/media/auto-j/microcontrollers-microprocessors/rz/rzg/hmi-mmpoc-videos/h264-vga-24.mp4;name=video4 \
+           http://resource.renesas.com/resource/lib/img/products/media/auto-j/microcontrollers-microprocessors/rz/rzg/hmi-mmpoc-videos/h264-hd-30.mp4;name=video5 "
 
 SRC_URI[video1.md5sum] = "bb827df771f78ca8b0b35a58d24ad3ae"
 SRC_URI[video1.sha256sum] = "a625e1b48aa5cae4f85eac9b073ef84259a42b7b7941a7c394d87b8e7da12283"
-- 
1.9.1

